index.js

```diff
@@ -199,13 +199,13 @@ var Base = Class.extend({
     }
 
     var pkSql = '';
-    if (primaryKeyColumns.length > 1) {
-      pkSql = this._handleMultiPrimaryKeys(primaryKeyColumns);
-
-    } else {
-
-      primaryKeyColumns[0] = primaryKeyColumns[0].name;
-      columnDefOptions.emitPrimaryKey = true;
+    if (primaryKeyColumns.length) {
```

Comment:
This is also an option and I do not think that your version is more succinct b/c it is ambiguous. Both are an option though, checking in the else tree is one and checking from above is one. Actually doing it on the else tree would be one comparison less to be processed in the optimistic case, so generally also a good option to go for, although branch prediction probably diminishs the effect anyway :)